convert.lua

```diff
@@ -50,9 +50,9 @@ tcp_len_f               = Field.new('tcp.len')
 function get_ip_src()
 	local ip_src = ip_src_f()
 	local ipv6_src = ipv6_src_f()
-	if ip_src then 
+	if ip_src then
```

Comment:
thx for these fix but please next time commit this in a dedicated commit, not to pollute the interesting modification ;)